core/src/main/resources/io/questdb/site/conf/server.conf

```diff
@@ -147,6 +147,10 @@ query.timeout.sec=60
 
 ################ Cairo settings ##################
 
```

Comment:
Why not set it to 0 to reflect the default value ? 
Are you planning to add he prop to pkg/ami/marketplace/assets/server.conf ?

Reply:
Ok, I set it to 0.
No I don't plan to change AMI config initially